pkg/kubelet/dockertools/manager.go

```diff
@@ -605,6 +605,7 @@ func (dm *DockerManager) runContainer(
 			MemorySwap: -1,
 			CPUShares:  cpuShares,
 		},
+		ShmSize:     1024 * 1024 * 64,
```

Comment:
I think it would be better to set default value in [`kube_docker_client`](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/dockertools/kube_docker_client.go#L128).
Are you OK with that? :)
If so, I could send a PR right away.

Moreover, I'll file an issue to engine-api to see whether they can set `ShmSize` field to be "omitempty".